solidity/app/compile.js

```diff
@@ -0,0 +1,71 @@
+const solc = require('solc')
+const fs = require('fs')
+const path = require('path')
+
+let lookupPaths = ['./', './contracts/', './contracts/examples/', './node_modules/']
+
+function solCompile (filename) {
```

Comment:
I wouldn't mind a few more basic comments throughout this PR, I know this is a once off tool, but it's hard to predict how long we'll need to maintain it and it would be helpful for any future maintainers.